src/transformers/tokenization_utils_base.py

```diff
@@ -2249,6 +2326,11 @@ def convert_added_tokens(obj: Union[AddedToken, Any], add_type_field=True):
             legacy_format=legacy_format,
             filename_prefix=filename_prefix,
         )
+        added_tokens_file = save_files[-1] + " and " if self.get_added_vocab() else None
```

Comment:
Cheeky - the " and " should go in the logger message. Reading this is v. confusing until after reading the line below.
```suggestion
        added_tokens_file = save_files[-1] if self.get_added_vocab() else None
```